tests/acceptance/features/apiNotification/notification.feature

```diff
@@ -263,9 +263,11 @@ Feature: Notification
       }
       """
     Examples:
-      | language | subject            | message                                          |
-      | de       | Neue Freigabe      | Alice Hansen hat textfile1.txt mit Ihnen geteilt |
-      | es       | Recurso compartido | Alice Hansen compartió textfile1.txt contigo     |
+      | language | subject            | message                                          | api      |
+      | de       | Neue Freigabe      | Alice Hansen hat textfile1.txt mit Ihnen geteilt | Graph    |
+      | de       | Neue Freigabe      | Alice Hansen hat textfile1.txt mit Ihnen geteilt | settings |
+      | es       | Recurso compartido | Alice Hansen compartió textfile1.txt contigo     | Graph    |
+      | es       | Recurso compartido | Alice Hansen compartió textfile1.txt contigo     | settings |
```

Comment:
@ScharfViktor I've refactored this like so ^ to include a few scenarios with settings API, if we don't need coverage for settings API I can remove the related code.

Reply:
fine, thanks. Since we have settings endpoint it's good to have a couple of tests for it